kdbg/sourcewnd.cpp

```diff
@@ -22,6 +22,7 @@
 #include <algorithm>
 #include "mydebug.h"
 
+#include <QTimer>
```

Comment:
Please follow the usual sorting of headers. This one is probably best located between `QMouseEvent` and `kiconloader.h`